### PR TITLE
Installation and readme

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Please review my PR for README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,190 @@
...
+@import url(https://fonts.googleapis.com/icon?family=Material+Icons);
+@import url(./components/components.css);
+```
+
+### Components that Grand UI provides are:
+1. [Avatar](#avatar)
+1. [Alerts](#alerts)
+1. [Badge](#badge)
+1. [Button](#button)
+1. [Card](#card)
```
Comment (dev-prabhat) [inline README.md:40]: Index number is same for all components

File: app.js [hunk 2]
```diff
@@ -20,4 +20,31 @@ arrayOfListItems.forEach((element)=>{
...
+    activeTab(componentList,arrayOfListItems[7]);
+}else if(value === "text-utilities"){
+    activeTab(componentList,arrayOfListItems[8]);
+}else if(value === "lists"){
+    activeTab(componentList,arrayOfListItems[9]);
+}else if(value === "navbar"){
+    activeTab(componentList,arrayOfListItems[10]);
+}else{
+    activeTab(componentList,arrayOfListItems[1]);
+}
```
Comment (zoef-yt) [inline app.js]: Use switch-case it would be better than so many if-else.

File: getting-started/documentation.html [hunk 3]
```diff
@@ -33,7 +32,7 @@
         <ul class="list">
             <div class="list-tile font-primary font-x-large fw-600">Getting Started</div>
             <li class="list-item">
-                <a class="btn btn-link-secondary btn-focus-primary w-100 justify-content-start font-large p-y-0_5">Installation</a>
+                <a class="btn btn-link-secondary btn-focus-primary w-100 justify-content-start font-large p-y-0_5" href="#installation">Installation</a>
             </li>
         </ul>
         <p> </p>
```
Comment (zoef-yt) [inline getting-started/documentation.html]: This is an empty tag that can be removed.

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Overall the Documentation in the readme file is good and detailed description of each components

**Review by zoef-yt — APPROVED** [review]
The code looks good. You can merge with the mentioned changes. Nice work on the readme. Good job.

File: getting-started/documentation.html (hunk 3 above)
Comment (bhushannaikaroskar) [reply getting-started/documentation.html]: ok sure

**Review by dev-prabhat — APPROVED** [review]

**Pull request merged**